components/swatch/stories/swatch.test.js

```diff
@@ -19,11 +19,46 @@ export const SwatchGroup = Variants({
 			testHeading: "Full rounding",
 			rounding: "full"
 		},
+		{
+			testHeading: "Light border",
+			borderStyle: "lightBorder",
+		},
+		{
+			testHeading: "No border",
+			borderStyle: "noBorder",
+			rounding: "none",
+		},
+		{
+			testHeading: "Rectangle",
+			isRectangle: true,
+		},
+		{
+			testHeading: "Gradient",
+			isGradient: true,
+			gradient: "linear-gradient(to right, rgba(0, 0, 0, 88%), rgb(174, 216, 230))",
+			swatchColor: "rgba(174, 216, 230, 0.3)",
+		},
+		{
+			testHeading: "Image",
+			isImage: true,
+			imageUrl: "example-ava@2x.png",
+		},
+		{
+			testHeading: "Mixed value",
+			isMixedValue: true,
+		},
 	],
 	stateData: [
+		{
+			testHeading: "Disabled",
+			isDisabled: true,
+		},
 		{
 			testHeading: "No color",
-			swatchColor: undefined
+			swatchColor: undefined,
```

Comment:
I could also just add `borderStyle: "default"` to the no-color test state. That would override all borders, which seems like a somewhat acceptable compromise. However, the trade off is the light border story, in the no-color state, will have the default border. 

Is this a question for design?

Reply:
We did talk briefly in our group pairing about this test. I didn't opt for the `ignore: ["No border"]` option we mentioned, and went with the `borderStyle: "default"` in the no-color test case. Let me know what you think of my reasoning.

Looking ahead to swatch in S2, the empty swatch does have the default border, and none of the examples for swatch or swatchgroup have the noBorder option. I think that noBorder option in S1 is really what is throwing off this no-color test case. 

I could definitely just pull it out of `stateData`, and move it to `testData`. Does it make more sense there? I'm honestly not sure. Since this "noColor" option is also triggered if the swatch color is transparent, maybe we should move it to the test cases.

<img width="424" alt="Screenshot 2024-09-10 at 2 47 15 PM" src="https://github.com/user-attachments/assets/9895e86a-aee4-4426-b616-ce93cdb9c16f">

[In this commit](https://github.com/adobe/spectrum-css/pull/2925/commits/cac56c3ab21998396e31bf2e1f8e166388ea455b), I favored a little preparation for S2 and to avoid the `lightBorder` and `noBorder` situations: 
<img width="100" alt="Screenshot 2024-09-10 at 2 20 16 PM" src="https://github.com/user-attachments/assets/21c26715-d88c-47d8-8003-3bba5891d2f4">
<img width="112" alt="Screenshot 2024-09-10 at 2 19 40 PM" src="https://github.com/user-attachments/assets/764c3a89-0fa7-441b-94d6-3cff6fceb8d3">

What do you think though? Maybe this is something to bring up with the larger group?